client/my-sites/checkout/checkout-thank-you/redesign-v2/masterbar-styled/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Global, css } from '@emotion/react';
```

Comment:
I see, it seems we want to rewrite it since this file is also [close to this one](https://github.com/Automattic/wp-calypso/blob/update/plans-congrats/client/my-sites/checkout/checkout-thank-you/marketplace/masterbar-styled/index.tsx).

Will we remove those reference files in a follow-up PR after ensuring this one works as expected?

Reply:
> Will we remove those reference files in a follow-up PR after ensuring this one works as expected?

Yeah we will try to keep it down to one file in a future PR.
Issue tracking this :https://github.com/Automattic/dotcom-forge/issues/2963